dev-vcs/fuel/fuel-1.0.1.recipe

```diff
@@ -0,0 +1,88 @@
+SUMMARY="A Qt Fossil GUI"
+DESCRIPTION="Fuel is cross-platform GUI front-end for the excellent Fossil SCM \
+tool written in Qt. Fuel runs on Windows, OSX as well as various flavors of \
+Unix including Linux and FreeBSD."
+HOMEPAGE="https://fuel-scm.org/fossil/home/"
+COPYRIGHT="2018 Kostas Karanikolas"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://fuel-scm.org/files/releases/fuel-$portVersion.tar.gz"
+CHECKSUM_SHA256="4e463cb9f3cf9c31c7ae95ce76b558502c0b07e1dd4109da714997f0dc9a65db"
+PATCHES="fuel-$portVersion.patchset"
+SOURCE_FILENAME="fuel-$portVersion.tar.gz"
```

Comment:
```diff
 CHECKSUM_SHA256="4e463cb9f3cf9c31c7ae95ce76b558502c0b07e1dd4109da714997f0dc9a65db"
-PATCHES="fuel-$portVersion.patchset"
 SOURCE_FILENAME="fuel-$portVersion.tar.gz"
+PATCHES="fuel-$portVersion.patchset"
 ADDITIONAL_FILES="fuel.rdef.in"
```
[HaikuPorter-Guidelines#ordering](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#ordering)
Thanks :)

Reply:
ok done.. thanks

Reply:
I was wondering why I had attached my comment to the line which had `SOURCE_FILENAME`...
I just found the answer: I had noticed that we could remove that line instead of moving `PATCHES`.. but then I must have forgotten we could remove `SOURCE_FILENAME`, so I ended editing the comment the way I had planed to do it before noticing `SOURCE_FILENAME` could be removed.
```diff
-SOURCE_FILENAME="fuel-$portVersion.tar.gz"
```
I'm sorry about that. Could you remove the line with the `SOURCE_FILENAME` ?
Thanks again :)

BTW, the reason why we can remove this is because this is the original name of the archive, obviously.
And we prefer to only use a `SOURCE_FILENAME` if it's required.

Reply:
ok.. done mate